x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/components/assigned.js

```diff
@@ -52,28 +46,37 @@ export class Assigned extends React.Component {
         </a>
       </EuiKeyboardAccessible>
     );
-    const master = (data.node_type === 'master') ? <span className="fa fa-star" /> : null;
+    const master =
+      data.node_type === 'master' ? <EuiIcon type="starFilledSpace" color="primary" /> : null;
     const shards = sortBy(data.children, 'shard').map(this.createShard);
     return (
-      <div
+      <EuiFlexItem
+        grow={false}
         className={calculateClass(data, initialClasses.join(' '))}
         key={key}
         data-test-subj={`clusterView-Assigned-${key}`}
         data-status={shardStats && shardStats.status}
       >
-        <div className="title">{name}{master}</div>
-        {shards}
-      </div>
+        <EuiFlexGroup gutterSize="xs">
+          <EuiFlexItem grow={false} className="monTitle eui-textNoWrap">
+            {name}
+            {master}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="s">{shards}</EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexItem>
     );
   };
 
   render() {
     const data = sortBy(this.props.data, sortByName).map(this.createChild);
     return (
       <td>
-        <div className="children">
+        <EuiFlexGroup wrap className="monChildren">
```

Comment:
This class could be more descriptive. Try to match the component you are in. `monAssigned__children` or `monAssignedChildren`?